<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Q_Eula</class>
 <widget class="QDialog" name="Q_Eula">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>639</width>
    <height>419</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>JPEGsnoop License Agreement</string>
  </property>
  <widget class="QDialogButtonBox" name="buttonBox">
   <property name="geometry">
    <rect>
     <x>540</x>
     <y>10</y>
     <width>81</width>
     <height>71</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Vertical</enum>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>520</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Welcome to JPEGsnoop!</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>50</y>
     <width>520</width>
     <height>261</height>
    </rect>
   </property>
   <property name="title">
    <string>Software Agreement:</string>
   </property>
   <widget class="QPlainTextEdit" name="plainTextEdit">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>500</width>
      <height>191</height>
     </rect>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QCheckBox" name="cbAgreed">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>230</y>
      <width>500</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>I have read and agree to the terms above</string>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>320</y>
     <width>520</width>
     <height>91</height>
    </rect>
   </property>
   <property name="title">
    <string>Software Updates:</string>
   </property>
   <widget class="QCheckBox" name="cbUpdate">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>500</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Periodically check for updates
You can always change this option later in Configuration</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>Q_Eula</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>566</x>
     <y>80</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
